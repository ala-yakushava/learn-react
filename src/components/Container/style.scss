@import "../../styles/theme";

.Container {
  width: 1200px;
  margin: 0 auto;
  padding: 0 60px;
}
