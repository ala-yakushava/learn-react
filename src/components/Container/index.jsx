import React from 'react';
import PropTypes from 'prop-types';

import './style.scss';

const Container = ({ children }) => {
  return (
    <div className="Container">
      { children }
    </div>
  );
}

Container.propTypes = {
  children: PropTypes.node,
}

export default Container;
