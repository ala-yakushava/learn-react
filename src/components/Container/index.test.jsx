import React from 'react';
import renderer from 'react-test-renderer';

import Container from './index';

describe('Components - Container', () => {
  test('should match snapshot', () => {
    const tree = renderer
      .create(
        <Container>
          <div>Content</div>
        </Container>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
