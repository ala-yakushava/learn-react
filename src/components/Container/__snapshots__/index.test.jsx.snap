// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components - Container should match snapshot 1`] = `
<div
  className="Container"
>
  <div>
    Content
  </div>
</div>
`;
